build.gradle

```diff
@@ -16,7 +16,7 @@ apply plugin: 'com.github.ben-manes.versions' // gradle dependencyUpdates -Drevi
 apply plugin: 'project-report' // gradle htmlDependencyReport
 
 sourceCompatibility = 1.6
-version = '0.13.0'
+version = '0.14.0'
```

Comment:
@tamirms would it make sense to make this 1.0.0 too?

Reply:
I'm not sure but I don't think the sdk versions have to match horizon's version

Reply:
they don't  -- I was just suggesting also to add extra emphasis that there have been breaking changes through the latest release -- but I'm okay with this :)

Reply:
For what is worth none of the sdks match horizon's version  :)

Reply:
I think semver allows breaking changes without incrementing the major release if the version is less than 1.0.0